eng/jacoco-test-coverage/pom.xml

```diff
@@ -51,6 +51,11 @@
       <artifactId>azure-ai-textanalytics</artifactId>
       <version>5.1.0-beta.2</version> <!-- {x-version-update;com.azure:azure-ai-textanalytics;current} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-analytics-monitorexporter</artifactId>
```

Comment:
Just confirming, you're planning to change this to opentelemetry-exporter-azuremonitor

Reply:
Yes, this is updated.